package.json

```diff
@@ -91,6 +91,7 @@
 		"instantsearch.js": "^3.6.0",
 		"isomorphic-fetch": "^3.0.0",
 		"js-cookie": "^2.2.1",
+		"jshint": "^2.13.1",
```

Comment:
I don't think this should be added to dependencies?

Reply:
Right, its a dev dependency.